FirebasePerformance.podspec

```diff
@@ -42,7 +42,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
 
   s.public_header_files = base_dir + 'Sources/Public/FirebasePerformance/*.h'
 
-  preprocessor_definitions = 'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ' + 'FIRPerformance_LIB_VERSION=' + String(s.version)
+  preprocessor_definitions = 'FIRPerformance_LIB_VERSION=' + String(s.version)
```

Comment:
I think line 32 can also be removed.

The below files would not be required anymore.
`'ProtoSupport/**/*.[mh]'`

Reply:
Good catch! Done.